packages/grid/x-data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -339,6 +339,26 @@ describe('<DataGrid /> - Selection', () => {
       expect(checkbox).to.have.attribute('tabindex', '0');
       expect(checkboxCell).to.have.attribute('tabindex', '-1');
     });
+
+    it('should select/unselect all rows when pressing space', async () => {
```

Comment:
No need for asnyc here

```suggestion
    it('should select/unselect all rows when pressing space', () => {
```

Reply:
Will be corrected in #3675